templates/zerver/integrations.html

```diff
@@ -230,6 +230,12 @@
             <span class="integration-label">Travis CI</span>
          </a>
        </div>
+       <div class="integration-lozenge integration-yo-app">
+         <a class="integration-link integration-yo-app" href="#yo-app">
+            <img class="integration-logo" src="/static/images/integrations/logos/yo-app.png" alt="Yo App logo" />
+            <span class="integration-label">Yo App</span>
+         </a>
+       </div>
```

Comment:
This should be sorted alphabetically, so please move the Yo integration to just above zendesk (Travis is out of place)